src/gui/viewer.cpp

```diff
@@ -390,6 +394,32 @@ void ViewerMainWindow::reload_file() {
 	document()->load_file(document()->path().c_str());
 }
 
+void ViewerMainWindow::import_file() {
+	auto filename = QFileDialog::getOpenFileName(this, tr("Open Image"), QString(), "JPG Files (*.jpg)").toUtf8();
+
+	//	ImageImportDialog dialog(this);
+	//	dialog.exec();
+
+	if (filename.isEmpty())
+		return;
+
+	auto image = imglib::image_begin(filename);
```

Comment:
`image` needs to get null checked. `image_begin` may return null if the file does not exist.

Reply:
Showing an error dialog now if the `image` is null.